express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const gradesArr = [];
+  for (const key in grades) {
+    gradesArr.push(grades[key]);
+  }
+  res.json(gradesArr);
+});
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  res.status(201).json(grades);
```

Comment:
Here we just want to send the new thing the client created, instead of the whole data model.